src/components/Modals/ExpertOverviewModal.module.scss

```diff
@@ -22,3 +22,182 @@
     }
   }
 }
```

Comment:
It would also be fine to use the [tooltip from antd](https://ant.design/components/tooltip/) for example. That way you would save a lot of work and ensure future maintainability.